src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -297,6 +297,11 @@ def get_sonic_version_info():
 
     return data
 
+def get_sonic_version_file():
+    if not os.path.isfile(SONIC_VERSION_YAML_PATH):
+        return None
```

Comment:
Does all the current callers of this API, handle a return value of "None"  ? If this is a mandatory file, we could raise an exception.

Reply:
There is only one caller (https://github.com/Azure/sonic-utilities/pull/1266). I'd say let's leave it to the caller to decide. Python will raise exception anyway.